docs/guide/management/apm.txt

```diff
@@ -1,3 +1,106 @@
 ===
 APM
 ===
+
+Application Performance Monitoring support is a driver
+feature that allows monitoring services to hook into the
+driver in a forward compatible and stable way. The API is
```

Comment:
Not required for lgtm, but would be a nice to have:

"Application Performance Monitoring support is a driver feature that allows monitoring services to hook into the
driver in a **forward compatible** and stable way."

should read 

"Application Performance Monitoring support is a driver feature that allows monitoring services to hook into the driver in a **forward-compatible** and stable way."

To fix:
* Change `forward compatible` to `forward-compatible`

Reply:
Please ignore the above comment as it involves a readability change rather than a code/technical change